it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartControllerISpec.scala

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmrc.agentsubscriptionfrontend.controllers
+
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentType, _}
+import uk.gov.hmrc.agentsubscriptionfrontend.stubs.AuthStub
+import uk.gov.hmrc.agentsubscriptionfrontend.support.SampleUsers._
+
+class StartControllerISpec extends BaseControllerISpec {
+
+  private lazy val controller: StartController = app.injector.instanceOf[StartController]
+
+  "context root" should {
+    "redirect to start page" in {
+      val result = await(controller.root(FakeRequest()))
+
+      status(result) shouldBe 303
+      redirectLocation(result).head should include ("/start")
+    }
+  }
+
+  "start" should {
+    "not require authentication" in {
+      AuthStub.userIsNotAuthenticated()
+
+      val result = await(controller.start(FakeRequest()))
+
+      status(result) shouldBe 200
+    }
+
+    "be available at /agent-subscription/start" in {
```

Comment:
I don't think this test actually tests that start is available at this particular URL, because it calls the controller method directly instead of using `route()`.

So far we have been leaving tests of the URLs for end-to-end testing so I suggest just changing the description rather than the implementation of the test.